kotlinpoet/src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -690,7 +690,7 @@ class FunSpecTest {
 
     assertThat(builder.build().toString()).isEqualTo("""
       |@kotlin.jvm.JvmStatic
-      |internal fun staticMethod() {
+      |internal fun staticMethod(): kotlin.Unit {
```

Comment:
Does explicit API mode require `Unit` on non-public methods?

Reply:
I don't think so but it's probably easier to just emit it always. Had a similar discussion in #926 with @JakeWharton

Reply:
Ah, cool, agree.